src/can.rs

```diff
@@ -0,0 +1,498 @@
+pub use embedded_hal_can::{self, Filter, Frame, Id, Receiver, Transmitter};
+
+use crate::gpio::gpioa;
+use crate::gpio::AF9;
+use crate::rcc::APB1;
+use crate::stm32;
+use arrayvec::ArrayVec;
+use nb;
+use nb::Error;
```

Comment:
Could also be written as

```suggestion
use nb::{self, Error};
```